### PR TITLE
feat: add cookie banner

### PR DESCRIPTION
**Pull request by sauldeleon**
image

disables on scroll and on accept

### CODE REVIEW

**Review by paulaguijarro — CHANGES_REQUESTED** [review]
On the post's pages, the same post appears in the list of "related posts". It should be excluded.
image

File: packages/blog/public/locales/es/common.json [hunk 1]
```diff
@@ -20,5 +20,10 @@
   },
   "categories": {
     "all": "Todas"
+  },
+  "cookieBanner": {
+    "message": "Emoción Blanca usa cookies para asegurarse de la mejor experiencia posible en nuestra web.",
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/blog/public/locales/es/common.json]: Fixed! thanks!

File: packages/blog/public/locales/en/postPage.json [hunk 2]
```diff
@@ -5,6 +5,6 @@
     "author": "by {{name}}",
     "share": "Share on {{socialNetwork}}",
     "shareTitle": "Check this article at White Emotion: {{title}}",
-    "relatedPosts": "Related posts"
+    "sameCategoryPosts": "Also in this category..."
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/blog/public/locales/en/postPage.json:8]: Yeah, name is better but it has no relation with the content of the variable. This name is reserved for tag relationed posts, not category ones!

File: packages/blog/src/pages/post/[postId]/[postName]/index.tsx [hunk 3]
```diff
@@ -91,7 +91,7 @@ export const getServerSideProps: GetServerSideProps<PostPageProps | WithErrorPro
       category: post?.categories?.[0]?.code,
     })
     const [responseRelatedPost] = await Promise.all([relatedPostsRequest])
-    relatedPosts = responseRelatedPost
+    relatedPosts = responseRelatedPost?.filter(relatedPost => relatedPost.id !== post?.id)
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/blog/src/pages/post/[postId]/[postName]/index.tsx:94]: Changed to new correct name

File: packages/blog/src/components/post/PostPage/PostPage.tsx [hunk 4]
```diff
@@ -238,7 +238,7 @@ export function PostPage({ post, about, relatedPosts }: PostPageProps) {
 
       {relatedPosts && relatedPosts?.length > 0 && (
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/blog/src/components/post/PostPage/PostPage.tsx:239]: Changed to new correct name

File: packages/blog/src/pages/post/[postId]/[postName]/index.tsx [hunk 5]
```diff
@@ -84,20 +84,20 @@ export const getServerSideProps: GetServerSideProps<PostPageProps | WithErrorPro
...
   if (post?.categories?.length) {
-    const relatedPostsRequest = getLatestPosts({
+    const sameCategoryPostsRequest = getLatestPosts({
       locale: locale as AppLocales,
       category: post?.categories?.[0]?.code,
     })
-    const [responseRelatedPost] = await Promise.all([relatedPostsRequest])
-    relatedPosts = responseRelatedPost
+    const [responsesameCategoryPost] = await Promise.all([sameCategoryPostsRequest])
+    sameCategoryPosts = responsesameCategoryPost?.filter(sameCategoryPost => sameCategoryPost.id !== post?.id)
```
Comment (paulaguijarro) [inline packages/blog/src/pages/post/[postId]/[postName]/index.tsx:94]: what is the limit for the number of posts shown?

Comment (sauldeleon) [reply packages/blog/src/pages/post/[postId]/[postName]/index.tsx:94]: no limit at the moment! Adding one. I think 4 should be enough

File: packages/blog/public/locales/en/postPage.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (paulaguijarro) [reply packages/blog/public/locales/en/postPage.json:8]: I meant the text shown, I don't like the ellipses

**Review by paulaguijarro — APPROVED** [review]

File: packages/blog/public/locales/en/postPage.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/blog/public/locales/en/postPage.json:8]: removing ellipsis!

**Pull request merged**